bank/handler.go

```
@@ -305,3 +477,102 @@ func has(list []string, a string) bool {
+		InstructionIdentification: initiation.InstructionIdentification,
+		LocalInstrument:           initiation.LocalInstrument,
+		RemittanceInformation:     (*paymentModels.OBWriteDomesticResponse5DataInitiationRemittanceInformation)(initiation.RemittanceInformation),
+		SupplementaryData:         initiation.SupplementaryData,
+	}
+}
+
+func getDefaultConsent() *paymentModels.OBWriteDomesticConsentResponse5 {
+	var consent paymentModels.OBWriteDomesticConsentResponse5
+
+	data := []byte(`{
+		"Data": {
+		  "ConsentId": "58923",
+		  "Status": "Authorized",
+		  "CreationDateTime": "2017-06-05T15:15:13+00:00",
+		  "StatusUpdateDateTime": "2017-06-05T15:15:22+00:00",
+		  "ReadRefundAccount": "Yes",
+		  "Initiation": {
+			"InstructionIdentification": "ACME412",
+			"EndToEndIdentification": "FRESCO.21302.GFX.20",
+			"InstructedAmount": {
+			  "Amount": "165.88",
+			  "Currency": "GBP"
+			},
+			"CreditorAccount": {
+			  "SchemeName": "UK.OBIE.SortCodeAccountNumber",
+			  "Identification": "08080021325698",
+			  "Name": "ACME Inc",
+			  "SecondaryIdentification": "0002"
+			},
+			"RemittanceInformation": {
+			  "Reference": "FRESCO-101",
+			  "Unstructured": "Internal ops code 5120101"
+			}
+		  },
+		  "Debtor": {
+			"Name": "D Jones"
+		  }
+		},
+		"Risk": {
+		  "PaymentContextCode": "EcommerceGoods",
+		  "MerchantCategoryCode": "5967",
+		  "MerchantCustomerIdentification": "053598653254",
+		  "DeliveryAddress": {
+			"AddressLine": [
+			  "Flat 7",
+			  "Acacia Lodge"
+			],
+			"StreetName": "Acacia Avenue",
+			"BuildingNumber": "27",
+			"PostCode": "GU31 2ZZ",
+			"TownName": "Sparsholt",
+			"CountySubDivision": [
+			  "Wessex"
+			],
+			"Country": "UK"
+		  }
+		},
+		"Links": {
+		  "Self": "https://api.alphabank.com/open-banking/v3.1/pisp/domestic-payment-consents/58923"
+		},
+		"Meta": {}
+	  }`)
+
+	json.Unmarshal(data, &consent) //nolint
```

```
if err := json.Unmarshal(data, &consent); err != nil {
  panic(err)
}
```
let's do not ignore errors like this, it'll be a hell for someone to debug it if he updates this json and it's going to fail here